cmd/apko/cli/publish.go

```diff
@@ -37,25 +40,39 @@ in a keychain.`,
 		Example: `  apko publish <config.yaml> <tag>`,
 		Args:    cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return PublishCmd(cmd.Context(), args[0], args[1])
+			digest, err := PublishCmd(cmd.Context(), args[0], args[1])
+			if err != nil {
+				return err
+			}
+			if imageRefs != "" {
+				if err := os.WriteFile(imageRefs, []byte(digest.String()), 0666); err != nil {
```

Comment:
I originally had 0600 here, but since this runs as `root` and the final build step is `nonroot` I tweaked this.

We can switch the final step in the task to run as root, or `nosec` this.

Reply:
I would suggest `nosec` :)